src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -66,6 +66,8 @@ private void MainForm_Load(object sender, EventArgs e)
 		{
 			UpdateWindowTitle();
 
+			WindowSizeSubMenu.DropDownItems.AddRange(CreateWindowSizeFactorSubmenus(max_factor: 10).ToArray());

```

Comment:
Why return a list if you're going to convert it to an array at the only call-site?

And there will only ever be one call-site, so you should inline it or use a local method.